nx-X11/programs/Xserver/Xext/xf86bigfont.c

```diff
@@ -638,12 +638,12 @@ ProcXF86BigfontQueryFont(
 	        + (nCharInfos+1)/2 * 2 * sizeof(CARD16)
 	      : 0);
 	xXF86BigfontQueryFontReply* reply =
-	   (xXF86BigfontQueryFontReply *) ALLOCATE_LOCAL(rlength);
+	   (xXF86BigfontQueryFontReply *) calloc(1, rlength);
```

Comment:
In X.org xserver, there is commit 

```
commit 0b7b89fbac0b3865b2cf51295c68a5f4c7523f28
Author: Adam Jackson <ajax@redhat.com>
Date:   Mon Oct 6 14:19:30 2008 -0400

    xalloc+bzero -> xcalloc

```
... but it does not mean that X.org xserver had enough bzero()s in place, when they dropped (DE)ALLOCATE_LOCAL.